.docker/selfhost/schema.json

```diff
@@ -921,6 +905,22 @@
         }
       }
     },
+    "customerIo": {
```

Comment:
seems no change

Reply:
The order of auto-generation has changed and can only be executed once, otherwise the CI will not pass